src/model/__tests__/__snapshots__/ColumnConfiguration.spec.tsx.snap

```diff
@@ -1348,9 +1348,13 @@ exports[`ColumnConfiguration component should render all columns 64`] = `
 
 exports[`ColumnConfiguration component should render all columns 65`] = `
 <DocumentFragment>
-  <ul
-    class="no-bullet"
-  />
+  <section
```

Comment:
I noticed that this snapshot is for `Gene Ontology IDs` (because the `text-block` is now present after your changes) and since it is empty that means our dummy model data is lacking this. Do we want to take a note of these absences for a future date when we will populate the dummy data or do we want to fix as we go along?

Reply:
We're getting `modelData.json` from the backend and indeed it's missing GO terms. Will check with @LeonardoGonzales